helm/polaris/README.md

```diff
@@ -186,161 +186,161 @@ kubectl delete namespace polaris
+| persistence.eclipseLink.secret.name | string | `nil`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | The name of the secret to pull persistence.xml from. If not provided, the default built-in persistence.xml will be used. This is probably not what you want.                                                                                                                                                                                                                                                                                                                        |
+| persistence.type | string | `"relational-jdbc"`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | The type of persistence to use. Three built-in types are supported: in-memory, eclipse-link, relational-jdbc.                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
I'd rephrase it like this for clarity: `Three built-in types are available: "relational-jdbc", "in-memory", "eclipse-link". The in-memory type is not recommended for production use. The eclipse-link type is deprecated and will be unsupported in a future release.`